src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3507,11 +3501,6 @@ private ConversionOperatorDeclarationSyntax TryParseConversionOperatorDeclaratio
 
             ExplicitInterfaceSpecifierSyntax tryParseExplicitInterfaceSpecifier()
             {
-                if (this.CurrentToken.Kind == SyntaxKind.OperatorKeyword)
-                {
-                    return null;
-                }
```

Comment:
unnecessary.  the very next check bails if we don't have an identifiertoken, so if we have an operatorkeyword, that would always bail.